mesa/experimental/cell_space/cell.py

```diff
@@ -56,20 +57,24 @@ def __init__(
         """
         super().__init__()
         self.coordinate = coordinate
-        self._connections: list[Cell] = []  # TODO: change to CellCollection?
-        self.agents = []  # TODO:: change to AgentSet or weakrefs? (neither is very performant, )
+        self.connections: dict[str, Cell] = {}
```

Comment:
From my understanding of the code, in case of orthogonal grids, the key is a tuple not a str? So is the typing here correct or should it be Any?